src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -58,6 +59,17 @@ public Object clone() throws CloneNotSupportedException {
       Map<String,List<String>> awsDimensionSelectRegex;
       String help;
       boolean cloudwatchTimestamp;
+
+      List<String> allStats() {
```

Comment:
This name is misleading. Per the name I'd expect it to be the defaults if nothing is user specified, not copying in the configuration.